setup.py

```diff
@@ -56,13 +86,6 @@ def run_tests(self):
 [visualization docs]: https://docs.datadoghq.com/tracing/visualization/
 """
 
-# psutil used to generate runtime metrics for tracer
-install_requires = ["psutil>=5.0.0"]
-
-# include enum backport
-if sys.version_info[:2] < (3, 4):
```

Comment:
@majorgreys can you remind me why this is here?

Reply:
#1150 added an `class SpanTypes(Enum)` and we needed to include the backport. The logic here was to only install the backport for PY2.

Reply:
Ah, sorry, I'll add back, I did a search for `Enum34` and didn't find anything, wasn't thinking.